x-pack/test/apm_api_integration/tests/mobile/mobile_detailed_statistics_by_field.spec.ts

```diff
@@ -93,7 +93,8 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             environment: 'production',
             field: 'service.version',
           });
-          expect(isEmpty(response.currentPeriod)).to.be.equal(false);
+
+          expect(Object.keys(response.currentPeriod)).to.be.eql(['2.3', '1.1', '1.2']);
```

Comment:
What about:

```suggestion
          expect(Object.keys(response.currentPeriod)).to.be.eql(SERVICE_VERSIONS);
```

Reply:
You probably want to sort these, to make sure that any change in sorting doesn't break the test.

Reply:
Good point